csrc/ir_cloner.h

```diff
@@ -146,4 +146,20 @@ T* IrBuilder::clone(const T* src, IrCloner* ir_cloner) {
 template <typename T>
 NVFUSER_DEFINE_CLONE(Attribute<T>)
 
+template <typename T>
```

Comment:
Why put these functions here and not in fusion.h?

Reply:
Because `fusion.h` can not include `ir_cloner.h` (cyclic dependency), so I can not define them in `fusion.h`.